src/views/Restoration/RestorationBusinessName.vue

```diff
@@ -110,33 +123,6 @@ h2::before {
   content: counter(header-counter) '. ';
 }
 
-.value.name-request {
-  width: 100%;
-  min-width: 24rem;
-}
-
-.meta-container {
-  display: flex;
-  flex-flow: column nowrap;
-  position: relative;
-
-  > label:first-child {
-    font-weight: bold;
-  }
-}
-
-@media (min-width: 768px) {
-  .meta-container {
-    flex-flow: row nowrap;
-
-    > label:first-child {
-      flex: 0 0 auto;
-      padding-right: 2rem;
-      width: 12rem;
-    }
-  }
-}
-
 header p {
   padding-top:0.5rem
```

Comment:
It's not possible to do this with Vuetify spacing right? I searched last time everywhere for a way to do it but couldn't find any. I remember I asked Ethan about what to do with 15px, he told me just make it 16 which is pt-4.

Reply:
Absolutely: it would be helper class "pt-2".

But this selects all paragraphs in all headers, which may be less code. (I didn't write this.)

Reply:
I fixed the syntax on this line of code, so your comment no longer points to it ("Outdated").

Anyway, if you're OK with my answer then please resolve this conversation that you started,

Reply:
@JazzarKarim OK to resolve this conversation?